twoops_tracker/py/twoopstracker/.env.template

```diff
@@ -0,0 +1,27 @@
+# Required
```

Comment:
Shouldn't this be in just `twoops_tracker` since it affects everything? i.e. both `docker` and `py`.